client/src/containers/FormBuilderLoader/createIdentifier.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Creates a dot-separated identifier for forms generated
+ * with schemas (e.g. FormBuilderLoader)
+ *
+ * @param props
+ * @returns {string}
+ */
+const createIdentifier = (props) => {
```

Comment:
I guess I'd prefer more defined params for helper functions like this... object is fine, but the nested name is weird and specific to `FormBuilder`